src/languageservice/services/yamlCompletion.ts

```diff
@@ -1130,25 +1168,30 @@ export class YamlCompletion {
           }
           case 'array':
             {
-              const arrayInsertResult = this.getInsertTextForArray(propertySchema.items, separatorAfter, insertIndex++, indent);
-              const arrayInsertLines = arrayInsertResult.insertText.split('\n');
-              let arrayTemplate = arrayInsertResult.insertText;
-              if (arrayInsertLines.length > 1) {
-                for (let index = 1; index < arrayInsertLines.length; index++) {
-                  const element = arrayInsertLines[index];
-                  arrayInsertLines[index] = `  ${element}`;
-                }
-                arrayTemplate = arrayInsertLines.join('\n');
-              }
+              const arrayInsertResult = this.getInsertTextForArray(
```

Comment:
refactor and fix indentation
use new `addIndentationToMultilineString` function